### PR TITLE
[FR DateTimeV2] Fix French datetime matches numbers

### LINKED ISSUE #1523: [FR DateTimeV2] French datetime matches numbers
For some reason DateTime Model in French matches numbers from 0 to 24
Example: "Je veux le numéro 3", `3` is matched as datetime when it should not.

### PR DESCRIPTION
**Pull request by Grey0202**
- This fixes #1523
- The single number now will not be recognized as Time
- Add some tests.

### CODE REVIEW

File: Specs/DateTime/French/DateTimeModel.json [hunk 1]
```diff
@@ -1781,5 +1781,341 @@
...
+  {
+    "Input": "Dix heures mercredi matin",
+    "Context": {
+      "ReferenceDateTime": "2019-08-12T00:00:00"
+    },
+    "Comment": "Not supported because of an extra result.",
+    "NotSupported": "dotnet, javascript, java, python",
+    "Results": [
+      {
+        "Text": "dix heures mercredi mati",
```
Comment (tellarin) [inline Specs/DateTime/French/DateTimeModel.json]: It should end in "matin".

File: Specs/DateTime/French/DateTimeModel.json [hunk 2]
```diff
@@ -1781,5 +1781,341 @@
...
+        }
+      }
+    ]
+  },
+  {
+    "Input": "Dix heures mercredi matin",
+    "Context": {
+      "ReferenceDateTime": "2019-08-12T00:00:00"
+    },
+    "Comment": "Not supported because of an extra result.",
```
Comment (tellarin) [inline Specs/DateTime/French/DateTimeModel.json]: What do you mean by "extra result"?

File: Specs/DateTime/French/SetExtractor.json [hunk 3]
```diff
@@ -33,13 +33,13 @@
     ]
   },
   {
-    "Input": "Je vais partir 9 lundis",
```
Comment (tellarin) [inline Specs/DateTime/French/SetExtractor.json]: Does "Je vais partir à 9 lundis" work?

Comment (Grey0202) [reply Specs/DateTime/French/SetExtractor.json]: "9 lundis" can be extracted but without parse result the same as the previous version.

Comment (tellarin) [reply Specs/DateTime/French/SetExtractor.json]: OK. Please add this case in the spec.

**Pull request merged**